### DISCUSSION TITLE
这里好像多了点儿什么？

### DISCUSSION THREAD

**Discussion by li-bing-peng** [gkd-kit/gkd#198] [General]
如图。
![微信图片_20231024162416](https://github.com/gkd-kit/gkd/assets/114422199/4db6681f-55cb-4033-97ff-8a28d3b7ae73)

**Comment by lisonge** [comment]
[debug.zip](https://github.com/gkd-kit/gkd/files/13112120/debug.zip)

试试这个测试版本呢，看看显示是否有效？

**Reply by li-bing-peng** [reply]
没了！太棒了！:-)

**Comment by lisonge** [comment]
https://github.com/gkd-kit/gkd/releases
